opensrp-maternity/src/main/java/org/smartregister/maternity/presenter/BaseMaternityRegisterActivityPresenter.java

```diff
@@ -162,7 +163,7 @@ public void startForm(@NonNull String formName, @Nullable String entityId, Strin
             form = model.getFormAsJson(formName, entityId, locationId, injectedFieldValues);
             // Todo: Enquire if we have to save a session of the outcome form to be continued later
             if (formName.equals(MaternityConstants.Form.MATERNITY_OUTCOME) || formName.equals(MaternityConstants.Form.MATERNITY_MEDIC_INFO)) {
-                interactor.fetchSavedMaternityOutcomeForm(entityId, entityTable, this);
+                interactor.fetchSavedMaternityOutcomeForm(MaternityUtils.convertFormNameToType(formName), entityId, entityTable, this);
```

Comment:
copy over `void fetchSavedPartialForm(@Nullable String formType, @NonNull String baseEntityId, @NonNull String entityTable);` to the `MaternityRegisterActivityContract` and remove `fetchSavedMaternityOutcomeForm`

Reply:
You should be fetching with form type not even type

Reply:
method renamed.